augur/refine.py

```diff
@@ -97,6 +97,7 @@ def register_arguments(parser):
     parser.add_argument('--output-tree', type=str, help='file name to write tree to')
     parser.add_argument('--output-node-data', type=str, help='file name to write branch lengths as node data')
     parser.add_argument('--timetree', action="store_true", help="produce timetree using treetime")
+    parser.add_argument('--use-fft', action="store_true", help="produce timetree using FFT for convolutions")
```

Comment:
FFT convolutions are only available for marginal likelihood calculations, right? If so, we might want to add a note about that in this help text.